src/main/groovy/org/cadixdev/gradle/licenser/tasks/LicenseTask.groovy

```diff
@@ -48,18 +57,39 @@ class LicenseTask extends DefaultTask {
     PatternFilterable filter
 
     @Input
-    String charset
+    abstract Property<String> getCharset()
 
     @Input
-    boolean skipExistingHeaders
+    abstract Property<Boolean> getSkipExistingHeaders()
 
     @InputFiles
     @SkipWhenEmpty
+    // @IgnoreEmptyDependencies (only in 6.8)
```

Comment:
Curious, does a missing annotation class break something at runtime?

Reply:
I suspect it doesn't -- but licenser currently builds against Gradle 6.4 and I haven't changed that.

Reply:
Might be worth changing, otherwise we will probably forget to uncomment this once updating the Gradle wrapper.